great_expectations/checkpoint/actions.py

```diff
@@ -912,11 +912,11 @@ def _basic_run(
 
         checkpoint_ge_cloud_id = None
         if self._using_cloud_context and checkpoint_identifier:
-            checkpoint_ge_cloud_id = checkpoint_identifier.id
+            checkpoint_ge_cloud_id = checkpoint_identifier.id or None
```

Comment:
We often interchange `""` and `None` in OSS code but tend to use a bit more explicitness in the Cloud backend. Since our schemas validate this data to be nullable UUIDs, passing forth empty strings results in runtime errors.

The approach here may also be changing `GXCloudIdentifier.id`'s type to a nullable string (instead of just a string).

Reply:
Good catch.
IMO having strings that could be empty is a confusing pattern.